kore/src/Kore/Step/BaseStep.hs

```diff
@@ -887,3 +860,528 @@ unwrapPredicateVariables
     => Predicate level (StepperVariable variable)
     -> m (Predicate level variable)
 unwrapPredicateVariables = traverse unwrapPatternVariables
+
+wrapUnificationOrSubstitutionError
+    :: Functor m
+    => ExceptT (UnificationOrSubstitutionError level variable) m a
+    -> ExceptT (StepError                      level variable) m a
+wrapUnificationOrSubstitutionError =
+    withExceptT unificationOrSubstitutionToStepError
+
+{- | Lift an action from the unifier into the stepper.
+ -}
+fromUnification
```

Comment:
`liftFromUnification`, maybe, to signal that this is a boilerplate-like transformation?